src/Drive.php

```diff
@@ -227,17 +253,18 @@ public function getFile(string $path)
     }
 
     /**
-     * get file as a file resource
+     * Download the file at the given path in the current drive and return the stream
```

Comment:
```suggestion
     * Download the file as a stream at the given path in the current drive
```

Reply:
I think the function will return the stream and the file will only be downloaded when the stream is read